Assets/MRTK/SDK/Experimental/MixedRealityKeyboard/MixedRealityKeyboardBase.cs

```diff
@@ -104,22 +104,40 @@ private enum KeyboardState
 
         #region MonoBehaviour Implementation
 
+#if !UNITY_EDITOR && UNITY_WSA
```

Comment:
Are we able to [use `WINDOWS_UWP` here](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#use-platform-dependent-compilation-to-ensure-the-toolkit-wont-break-the-build-on-another-platform)?

```suggestion
#if WINDOWS_UWP
```